linters/gokart/plugin.yaml

```diff
@@ -0,0 +1,30 @@
+version: 0.1
+lint:
+  definitions:
+    - name: gokart
+      files: [go]
+      runtime: go
+      package: github.com/praetorian-inc/gokart
+      environment:
+        - name: PATH
+          list: ["${linter}"]
+      run_from_root_file: go.mod
+      commands:
+        - name: lint
+          output: sarif
+          target: ${parent}
+          run: gokart scan -i ${workspace}/analyzers.yml -s ${target}
```

Comment:
After investigation, you were correct. This should be:
`run: gokart scan -i analyzers.yml -s ${target}`